tensorflow/python/kernel_tests/metrics_test.py

```diff
@@ -3541,7 +3541,7 @@ def testMultipleUpdatesWithWeights(self):
       _enqueue_vector(sess, weights_queue, [1.0])
       _enqueue_vector(sess, weights_queue, [1.0])
       _enqueue_vector(sess, weights_queue, [0.0])
-      _enqueue_vector(sess, weights_queue, [1.0])
```

Comment:
Why this change?

Reply:
It's the simplest way to exercise the new logic – the old code would fail with this case, since the weight rounds to 0.

Reply:
Truncate to 0, rather.